include/fiction/algorithms/simulation_sidb/simulation_parameter.hpp

```diff
@@ -0,0 +1,63 @@
+//
+// Created by Jan Drewniok on 24.11.22.
+//
+
+#ifndef FICTION_SIMULATION_PARAMETER_HPP
+#define FICTION_SIMULATION_PARAMETER_HPP
+
+#include "fiction/technology/constants.hpp"
+
+namespace fiction
+{
+struct simulation_parameter
+{
+     explicit simulation_parameter(const double relative_permittivity = 5.6, const double screening_distance = 5.0 * 1E-9, const double mu_minus = -0.32, const double a = 3.84 * 1E-10, const double b = 7.68 * 1E-10,
+                                            const double c = 2.25 * 1E-10) noexcept:
+            lat_a{a},
+            lat_b{b},
+            lat_c{c},
+            epsilon_r{relative_permittivity},
+            k{1.0 / (4.0 * 3.141592653 * fiction::physical_sim_constants::EPSILON * epsilon_r)},
```

Comment:
Unfortunately, mathematical constants like Pi are only available in C++20, but you could define them yourself via `constexpr long double PI = 3.14159265359`